src/components/table/table.scss

```diff
@@ -0,0 +1,86 @@
+@use "../../design-tokens/colors.yml";
+@use "../../design-tokens/font.yml";
+@use "../../design-tokens/sizes.yml";
+@use "../../mixins/ms";
+
+$table-border-color: colors.$gray-light;
+$table-border-thin: sizes.$edge_small solid $table-border-color;
+$table-border-thick: sizes.$edge_medium solid $table-border-color;
+
+/**
+ * Our base table component
+ *
+ * Since tables require specific markup using the BEM element class names can
+ * quickly become overwhelming. We make an exception from our BEM rules to
+ * directly style elements within `.c-table`s.
+ */
+
+/**
+ * 1. Tables may overflow horizontally. Keep this from breaking layouts while
+ *    ensuring content is viewable
+ *    In the future we may want to consider making it visually obvious when
+ *    tables are clipped.
+ */
+
+.c-table {
+  overflow-x: auto; /* 1 */
```

Comment:
Any reason why we wouldn't want this on our default `<table>` elements?

Reply:
Hmm, I suppose not... I can move that over

Reply:
done!